test/integration/prepIntegrationTests.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+#set -e
+#Usage: ./test/integration/prepIntegrationTests.sh <yourapikeyintheformofABCD:EFGH> <openwhis hostname> <openwhisk namespace> <api gatewaytoken>
+# Run from the incubator-openwhisk-client-js
+
+# Make sure wsk is installed; exit if wsk is not available.
+if ! type "wsk" > /dev/null; then
+  echo "Exiting program; wsk is not installed on the system. Cannot preform tests"
+  exit 1
+fi
+
+# Assert NODEJS6/NPM3 is default
+if [[ ! $(node --version) =~ v6.[0-9]*.[0-9]* ]]; then
```

Comment:
We should change the version check to support anything from 6 and above. Node.js 8 becomes the LTS version in a few months. https://github.com/nodejs/LTS